### DISCUSSION TITLE
External library: Generated thumbnail images listed in timeline

### DISCUSSION THREAD

**Discussion by eerojaaskelainen** [immich-app/immich#11692] [Q&A] [answered]
### The bug

I've noticed a sudden burst of preview images appear on timeline view. External library defined at /mnt/kuvat/ and Images shown are located in /mnt/kuvat/Immich/thumbs/. Image file names seem to be something like UUID, but they end with -thumbnail.webp or -preview.jpeg.

It seems that those are auto-generated preview images, but should not be shown in timeline. I'm not sure if those appeared after last update (from 1.109.2 to 1.111.0) because I introduced external library at the same time.

### The OS that Immich Server is running on

CasaOS

### Version of Immich Server

1.111.0

### Version of Immich Mobile App

1.111.0

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

```bash
...
```

**Comment by mmomjian** [comment, marked as answer]
This seems to be an issue with the way you configured the external libraries, but you haven’t provided any details on your setup.

**Comment by eerojaaskelainen** [comment]
@mmomjian You are absolutely right. I accidentally imported old Immich backup folder that I had nested in same import folder awhile ago.

Sorted it out, and rescan libraries - all OK. So my bad, app works flawlessly.
